sw/device/tock/chips/opentitan_common/src/lib.rs

```diff
@@ -0,0 +1,11 @@
+//! Implementations for generic LowRISC peripherals.
+
+#![feature(asm, concat_idents, const_fn, core_intrinsics)]
+#![feature(in_band_lifetimes)]
+#![feature(exclusive_range_pattern)]
```

Comment:
Consider avoiding `features` that you don't truly need, especially since I'm working to remove all of them from Tock.